WordPress/create-block-theme
# Update escaping function

## What?
Use `esc_attr_e` to escape HTML attributes of blocks.

## Why?
- To avoid warnings like: https://github.com/Automattic/themes/pull/8016#issuecomment-2288533682
- Use a Wordpress core escaping function specifically designed for that.

## How?
By adding a specific function for escaping HTML attributes of blocks.

## How to test:
- Export a theme with text in HTML attributes. Example `alt` property of Image blocks.
Check that the attributes are correctly escaped.
